source/UF/ConnectedTypes.lagda

```diff
@@ -0,0 +1,212 @@
+Ian Ray, 23rd July 2024
+
+We define connected types and maps (recall our convention that H-levels start
+at 0). We then explore relationships, closure properties and characterizations
+of interest pertaining to the concept of connectedness.
+
+\begin{code}
+
+{-# OPTIONS --safe --without-K #-}
+
+open import UF.FunExt
+open import UF.PropTrunc 
+
+module UF.ConnectedTypes (fe : Fun-Ext)
```

Comment:
Can you please indent like this instead?
```
module UF.ConnectedTypes 
        (fe : Fun-Ext)
        (pt : propositional-truncations-exist)
       where
```
Please do this in your other files too.

Reply:
For all modules? What if there is only one module param?

Reply:
Ah right, this should only need Fun-Ext now. You can also use a single line if there's only one assumption.

Reply:
The `where` should be on that same line.

Reply:
Well I do use pt in this file a fair bit because of all the inhabitedness. So I plan to still use it globally here. I was gonna fix it in my other files if necesary.

Reply:
But you can just use the 1-truncation? I'm not sure which is more natural.

Reply:
I mean I have shown they are equivalent but I think it's more clear to use the prop trunc. Would it be worthwhile to show the truncations-exist omplies prop-trunc-exists?